### PR TITLE
Lots of UI updates for better readability.

### PR DESCRIPTION
**Pull request by eggchan**
Add an alert component that can be utilized everywhere on the site. Create a focus functionality so that the page focuses on the alert when it is triggered.

### CODE REVIEW

File: src/AdminPage.js [hunk 1]
```diff
@@ -111,38 +112,33 @@ const AdminChild = () => {
         <meta
           name="description"
           content={
-            "Admin page for " + finalAdminEvent.eventname ?? "Admin page"
+            "Admin page for " + finalAdminEvent.eventname ?? "Untitled event"
```
Comment (rkaufman13) [inline src/AdminPage.js:115]: lmao wow good catch

File: src/AdminPage.js [hunk 2]
```diff
@@ -111,38 +112,33 @@ const AdminChild = () => {
...
           }
         />
       </Helmet>
       <div>
-        <h1>{finalAdminEvent.eventname ?? "Untitled event"}</h1>
+        <h1 className="d-inline">{finalAdminEvent.eventname ?? "Untitled event"}</h1>
+        {/* Curious on your thoughts on this */}
+        <span className={`p-2 ms-3 position-relative rounded statusTag ${noodIsActive ? "bg-success" : "bg-danger"}`}>{noodIsActive ? "ACTIVE" : "CLOSED"}</span>
```
Comment (rkaufman13) [inline src/AdminPage.js:122]: I didn't think I would like it but I like it

File: src/Alert.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from "react";
+import { Alert } from "react-bootstrap";
+import { ErrorIcon } from "./resources/ErrorIcon";
+
+export const Alerts = (props) => {
+  useEffect(() => {
+    // Focus on the alert the first time it's rendered--maybe there's a better way to do this and you can teach me
```
Comment (rkaufman13) [inline src/Alert.js:7]: I've never done this before but I googled it and apparently this is the way to do it :\

**Comment by rkaufman13** [comment]
looking good, if you're OK with my tweaks lmk and I'll officially approve

**Comment by eggchan** [comment]
> looking good, if you're OK with my tweaks lmk and I'll officially approve

Looks good to meeeee

**Review by rkaufman13 — APPROVED** [review]

**Pull request merged**